### PR TITLE
Comply with REUSE version 3.0 for licensing declaration.

### PR DESCRIPTION
**Pull request by apteryks**

### CODE REVIEW

File: srfi-1-reference.scm [hunk 1]
```diff
@@ -1,10 +1,6 @@
-;;; SRFI-1 list-processing library			-*- Scheme -*-
```
Comment (arthurgleckler) [inline srfi-1-reference.scm:1]: We can't remove Olin's license without his permission.  His license here isn't an MIT license, and it specifically forbids removing the license.  (The MIT license notice appears in the document, but not here.)

Comment (apteryks) [reply srfi-1-reference.scm:1]: Good catch!  I've contacted them to see if they'd agree to relicense this work as Expat (MIT).

Comment (apteryks) [reply srfi-1-reference.scm:1]: I haven't heard back from them yet, so in the meantime I've used a custom license (via the LicenseRef SPDX facility).

Comment (arthurgleckler) [reply srfi-1-reference.scm:1]: I now believe that I was wrong here.  As someone on the list pointed out, almost every SRFI, including SRFI 1, was changed to use the MIT license years ago.  The authors were contacted and approved the change.  So this change was reasonable.  Sorry about that.

Comment (apteryks) [reply srfi-1-reference.scm:1]: Could you point to this message on the list, or better, to Olin's original message saying so, if it was public on the list? I could register that link in the commit message for reference.

**Comment by arthurgleckler** [comment]
See [this message](https://srfi-email.schemers.org/srfi-announce/msg/2652023/).  It's the one that Philip McGrath pointed out.  It isn't specific to SRFI 1, but does specifically say that all authors at the time had agreed to using the MIT license — except for the authors of 5 and 24, who couldn't be reached.  David van Horn, who wrote the message, was a SRFI editor at the time.

File: srfi-1-reference.scm (hunk 1 above)
Comment (apteryks) [reply srfi-1-reference.scm:1]: I've left the original copyright notice and marked the file with the MIT SPDX license identifier.

**Pull request merged**